server/database/index.js

```diff
@@ -32,7 +32,7 @@ const savePost = (post) =>
   // connection.connect();I don't think we need this, but leaving it here for now??
   new Promise((resolve, reject) => {
     // attempt to avoid sql injection. Not sure if this is completely correct though
-    const postInsert = 'INSERT INTO posts(postId, text, img1, img2, img3) VALUES (DEFAULT, ?)';
+    const postInsert = 'INSERT INTO posts(postId, postText, img1, img2, img3, userId) VALUES (DEFAULT, ?)';
     // assuming <post> parameter is an object
     const insertValues = [post.text, post.img1, post.img2, post.img3, post.userId];
```

Comment:
shouldn't line 35 be const insertValues = [post.id, post.text, post.img1, post.img2, post.img3, post.userId];
so that they match up with the input values?

Reply:
line 37 rather

Reply:
no, the postId is handled by the "DEFAULT" in the (DEFAULT, ?) at the end of <postInsert>. Since postId is an AUTO_INCREMENT num that is an integer and initially starts at 1, and since we don't want the user to have to keep up with their number of posts, we want this value to always default to value mySQL has automatically assigned to it.

Reply:
cool, didn't know that, let the merger commence